core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -177,6 +177,11 @@ private SqlLibraryOperators() {
       ReturnTypes.cascade(ReturnTypes.explicit(SqlTypeName.VARBINARY),
           SqlTypeTransforms.TO_NULLABLE), null, OperandTypes.STRING, SqlFunctionCategory.STRING);
 
+  @LibraryOperator(libraries = {MYSQL})
+  public static final SqlFunction UNCOMPRESS = new SqlFunction("UNCOMPRESS", SqlKind.OTHER_FUNCTION,
+      ReturnTypes.cascade(ReturnTypes.explicit(SqlTypeName.VARCHAR),
+          SqlTypeTransforms.FORCE_NULLABLE), null,
+      OperandTypes.or(OperandTypes.STRING, OperandTypes.BINARY), SqlFunctionCategory.STRING);
 
```

Comment:
Why we declare to support a `STRING` params if there is no runtime implementation ?

Reply:
Reverted the change